command/vendored_plugins.go

```diff
@@ -0,0 +1,46 @@
+package command
+
+import (
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+
+	// Previously core-bundled components, split into their own plugins but
+	// still vendored with Packer for now. Importing as library instead of
+	// forcing use of packer init, until packer v1.8.0
+	exoscaleimportpostprocessor "github.com/exoscale/packer-plugin-exoscale/post-processor/exoscale-import"
```

Comment:
This change was also made in #10695 so we need to reconcile once merged but it should be straight forward.

Reply:
This looks great! Now that the Docker PR is merged, you can tweak this. Or I can, whatever you prefer.

Reply:
I rebased so all good here. Thanks for merging in the docker extraction PR.